components/org.wso2.identity.event.websubhub.publisher/src/main/java/org/wso2/identity/event/websubhub/publisher/service/WebSubEventPublisherImpl.java

```diff
@@ -133,57 +88,6 @@ private void makeAsyncAPICall(SecurityEventTokenPayload eventPayload, EventConte
                 });
     }
 
-    private void makeTopicMgtAPICall(String topic, String webSubHubBaseUrl, String operation, String tenantDomain)
```

Comment:
Is this a new publisher or the one being used for existing asgardeo events ? If it's latter we can't do any edits there

Reply:
This is the new one
and it's not removed, rather passed to a different file